openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1363,6 +1363,9 @@ This specification defines the following Credential Issuer Metadata parameters:
           * `alt_text`: OPTIONAL. String value of the alternative text for the logo image.
       * `description`: OPTIONAL. String value of a description of the Credential.
       * `background_color`: OPTIONAL. String value of a background color of the Credential represented as numerical color values defined in CSS Color Module Level 37 [@!CSS-Color].
+      * `background_image`: OPTIONAL. Object with information about the background image of the Credential. The following non-exhaustive set of parameters MAY be included:
```

Comment:
Should this have the 'defined by this specification' text as per comments on https://github.com/openid/OpenID4VCI/pull/238 that objected to me removed that text? :-)

Reply:
```suggestion
      * `background_image`: OPTIONAL. Object with information about the background image of the Credential. At least the following parameter MUST be included:
```